src/main/resources/assets/api-docs.yaml

```diff
@@ -1738,6 +1729,28 @@ paths:
           description: DAR or User Not Found
         500:
           description: Internal Server Error.
+  /api/dar/draft:
+    post:
+      summary: Create a Draft Data Access Request
+      description: Returns the created partial DAR
+      tags:
+        - Data Access Request
+      responses:
+        200:
+          description: Returns the created partial Data Access Requests
+        500:
+          description: Internal Server Error.
+  /api/dar/draft/{referenceId}:
```

Comment:
Same issue as the endpoint prior, endpoint is on V2 and already as documentation present.